### DISCUSSION TITLE
Getter Classes for Calculated Value Types

### DISCUSSION THREAD

**Discussion by Cweber-JC** [pimcore/pimcore#9290] [Q&A / Support]
Hello Everybody, 

iam currently running Pimcore 6.7.1 on Windows. Iam trying to create a class in data objects that combines the input of two different number classes. 
To do that iam trying to replicate the following steps in Pimcore doc:
https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Data_Types/Calculated_Value_Type.html
But in the doc it says to use "Getter methods on the object class that are generated as usual" when working with the PHP API and i can't find the usual getter methods anywhere. 

Does anybody know where to find the usual getter classes? And has anybody experience by working with the Calcutated Value Type?

I also feel like the working with Windows could also be a problem... 

Thank you for any suggestions and answers.

**Comment by fashxp** [comment]
Hi, 
have a look at the php classes that are generated at `var/classes/DataObject` for all classes. 

BR
Christian
